students/Adil/iteratorActivity.py

```diff
@@ -0,0 +1,50 @@
+#usr/bin/env python3
+
+"""
+returns the sequence of the numbers from zero to 4
+
+"""
+class IterateMe_1:
+
+    def __init__(self, stop=5):
+        self.current = 0
+        self.stop = stop
+    def __iter__(self):
+        return self
+    def __next__(self):
+        if self.current < self.stop:
+            self.current += 1
+            return self.current
+        else:
+            raise StopIteration
+
+class IterateMe_2:
+
+    def _init_(self, start=0, stop=5, step=1):
```

Comment:
This needs two `_` to work. Please try running your code to make sure it works first.